packages/edit-site/src/components/global-styles/font-sizes/font-size.js

```diff
@@ -46,6 +46,10 @@ function FontSize() {
 
 	const [ globalFluid ] = useGlobalSetting( 'typography.fluid' );
 
+	if ( ! origin || ! slug ) {
```

Comment:
Oh nice! I promise I didn't steal this for https://github.com/WordPress/gutenberg/pull/65811

😆